test/hotspot/jtreg/runtime/cds/appcds/sharedStrings/FlagCombo.java

```diff
@@ -26,6 +26,7 @@
  * @test
  * @summary Test relevant combinations of command line flags with shared strings
  * @requires vm.cds.archived.java.heap & vm.hasJFR
+ * @requires vm.gc == null
```

Comment:
Here and later: feels like these changes can be forked out into a separate testbug-only PR? Igor did quite a few touchups already for the tests that set their own options.

Reply:
I will move most of the test file changes into a separate RFE. [JDK-8272348](https://bugs.openjdk.java.net/browse/JDK-8272348): Update CDS tests in anticipation of JDK-8270489